pkgs/servers/sunshine/default.nix

```diff
@@ -59,11 +60,11 @@ stdenv.mkDerivation rec {
   ui = buildNpmPackage {
     inherit src version;
     pname = "sunshine-ui";
-    npmDepsHash = "sha256-k8Vfi/57AbGxYFPYSNh8bv4KqHnZjk3BDp8SJQHzuR8=";
+    npmDepsHash = lib.fileContents ./npm-hash;
```

Comment:
Please inline this, it shouldn't be hard with the unique variable

Reply:
Done.